src/client/components/ProductTile/ProductTile.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { getProduct } from '../../services/http';
+import Prices from "./Prices";
+import Swatches from './Swatches';
+import currencies from '../../static/currencies';
+
+const ProductTile = () => {
+  const [product, setProduct] = useState();
+  const { title, imageSrc, imageAlt, variationProducts, variationAttributes } = product || {};
+  
+  useEffect(() => {
+    getProduct("25518058M", { expand: "variations,images,prices" }).then(res => {
```

Comment:
Were you thinking to use actions/middleware/redux as alternative to making API calls directly in components?